dashbuilder/dashbuilder-client/dashbuilder-cms-client/src/main/java/org/dashbuilder/client/cms/screen/transfer/DataTransferView.java

```diff
@@ -55,6 +54,8 @@
     private Elemental2DomUtil elem2Dom;
     private Event<NotificationEvent> workbenchNotification;
     private PathUrlFactory pathUrlFactory;
+    
+    @Inject PagesTable pagesTable;
```

Comment:
Are you using this variable? 
Can it be injected on the constructor just like the others?